bluemira/codes/_freecadapi.py

```diff
@@ -73,6 +73,29 @@
 WORKING_PRECISION = 1e-5
 MIN_PRECISION = 1e-5
 MAX_PRECISION = 1e-5
+
+# ======================================================================================
+# Error catching
+# ======================================================================================
+
+
+def catch_caderr(raise_error):
```

Comment:
This argument should be an error type, from how it's named I thought it was meant to be a function. Maybe `new_error_type` would be a clearer name?